actions_helper/main.py

```diff
@@ -0,0 +1,48 @@
+import click
+
+from actions_helper.commands.update_dependency import get_packages_info, update_package
+from actions_helper.common.github_helpers import check_and_push_changes
+from actions_helper.common.utils import parse_bool
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command(name="dependency-update")
+@click.option("--dry-run", default="false", type=str)
```

Comment:
I was surprised that click doesn't have a type or parser for this. I guess you already looked for it as well, I also couldn't find anything in my quick search. Your `parse_bool` utils function is good in this case.